wiki/Internet_Relay_Chat/en.md

```diff
@@ -1,28 +1,25 @@
----
-needs_cleanup: true
----
-<!-- TODO: awkward terminology, some parts link to dated info, client-specific help should be updated with modern clients -->
-
 # Internet Relay Chat
 
-The [Internet Relay Chat](https://en.wikipedia.org/wiki/Internet_Relay_Chat), also known as IRC, is a well established standardized protocol for chatting with numerous clients available to connect with.
+*For further information, see [Internet Relay Chat (Wikipedia)](https://en.wikipedia.org/wiki/Internet_Relay_Chat "Wikiepdia")*
+
+The **Intenet Relay Chat** (**IRC**) is an application layer protocol for chatting with numerous clients available to connect with. Although not developed by/for osu!, it is an integral part of its infrastructure and to community [bots](/wiki/Bot_account).
 
 ## osu!Bancho
 
-osu!Bancho offers a gateway for IRC (in-game chat) access. You can connect with your own client and chat with other people without using the game client. Take note that this IRC protocol is modified therefore do not expect your client's features to work properly.
+*Notice to osu!Bancho users: this IRC protocol is modified, and thus it is not guaranteed that all of a clients' features will work properly on it. (E.g., [HexChat](https://hexchat.github.io/))*
 
-**Notice: [HexChat](https://hexchat.github.io/) is known to have problems with osu!'s IRC** ([bug report of HexChat's GitHub](https://github.com/hexchat/hexchat/issues/818)), consider using another client if it bothers you.
+*osu!Bancho* (sometimes shortened to just *Bancho*) is a modified version of the IRC written by [peppy](https://osu.ppy.sh/users/2) that offers a gateway for in-game chat access; designed to allow users to connect with their own client and chat with other people without the use of the osu! game client.
 
-## How to connect
+## Connection
 
-Once you have a client, you need to set up the server settings.
+Users wishing to set up a client with osu!Bancho must first set up its sever settings after creating an initial client. The information needed for these settings is listed below:
 
-- **Server:** `irc.ppy.sh`
-- **Port:** `6667` (default)
-- **Username:** Your osu! username (replace spaces to underscores)
-- **Password:** Get it at [IRC Authentication](https://osu.ppy.sh/p/irc).
+- Server: `irc.ppy.sh`
+- Port: `6667` (default)
+- Username: Client creator's osu! username (replace spaces to underscores) (e.g., `beppy master 1000` becomes `beppy_master_1000`)
+- Password: (Obtainable at the [IRC Authentication](https://osu.ppy.sh/p/irc) page).
 
-*Your IRC password is different from your account password. **Do not share it to others**.*
+*Warning: Every users' IRC password is different from their account password. **Do not** share this password with others.*
```

Comment:
this reads worse than original.

Reply:
Assuming you mean that it should be "separate" and not "different"? Or just that it sounds clunky?

Reply:
Written in first person sounds better. "their" "this" us weird.

that said there's an issue in the original "to others" should be "with others".